doc/nrf/ug_gzll.rst

```diff
@@ -7,22 +7,15 @@ Gazell Link Layer
    :local:
    :depth: 2
 
-.. gzll_intro_start
-
-Gazell is a protocol for setting up a robust wireless link between a single Host and up to eight Devices in a star network topology.
-It is designed to minimize power consumption in power-sensitive wireless desktop products and is also suitable for a range of other wireless applications.
-
-.. gzll_intro_end
-
-To minimize the power consumption of the power-sensitive peripheral devices, Gazell uses the central hub (host side) with its more relaxed power constraints to keep the link open while the peripheral devices can sleep and save on power consumption.
+To minimize the power consumption of the power-sensitive peripheral devices, Gazell uses the central hub (Host side) with its more relaxed power constraints to keep the link open while the peripheral devices can sleep and save on power consumption.
 A typical example of this is a wireless mouse communicating with a USB dongle that is inserted into a computer.
 
 Gazell provides a switching and synchronization scheme that reduces interference and provides wireless coexistence features, enabling high throughput and low latency.
 
 Features
 ********
 
-Gazell provides the following features:
+Gazell Link Layer provides the following features:
 
 * Support for star network topology with one Host and up to eight Devices.
```

Comment:
Does it make sense to add a link to Gazell modes here, where the star network is described?
Also, there, the introduction in lines 75-78 reads like bullet points. Would it make sense to describe first what Gazell modes are, how many are available, which are supported, and so on?

Reply:
Regarding Q1, yes I think that might be a good idea.
Q2, I have re-written lines 75-78 so that they are more complete sentences, rather than like bullet point.
Q3. Do you mean that in the Gazell modes section, starting on line 75, there should be more information about what Gazell modes are, etc., or should that information be described here in the Features section?

Reply:
Q3 refers to THAT section...